src/server/session.ts

```diff
@@ -404,7 +399,7 @@ module ts.server {
             var position = compilerService.host.lineColToPosition(file, line, col);
             var quickInfo = compilerService.languageService.getQuickInfoAtPosition(file, position);
             if (!quickInfo) {
-                throw Errors.NoContent;
+                return undefined;

```

Comment:
this will not send a result  to the client. i would send a {}; so that the client will get a message at least.